ksql-parser/src/main/java/io/confluent/ksql/parser/AstBuilder.java

```diff
@@ -1164,12 +1179,21 @@ public Node visitColumnReference(SqlBaseParser.ColumnReferenceContext context) {
         throw new InvalidColumnReferenceException("Field " + columnName + " is ambiguous.");
```

Comment:
not exactly related to this patch but the error message here is incorrect. Should be something like ```"Field" + columnName + " is present in neither the left nor right expression"